### ISSUE TITLE
Bendopnea

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10180]
**Preferred term label:**
Bendopnea

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C4087267 but our cardiologist explains that the shortness of breath is not instant and defines as "Dyspnea as a result of bending forward while not holding one's breath; dyspnea or shortness of breath onset within 30 seconds of bending forward"

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cardiomyopathies and pulmonary artery hypertensions.

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000013
- label: Bendopnea
- parents: HP:0002094
- pmid: PMID:31543202
- def: Dyspnea as a result of bending forward while not holding one's breath; dyspnea or shortness of breath onset within 30 seconds of bending forward.
- comment: Bendopnea is mediated by increased ventricular filling pressure during bending, which exacerbates an already high filling pressure in patients with heart failure.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10180
- synonyms:
